runtime/doc/diagnostic.txt

```diff
@@ -378,15 +378,25 @@ Lua module: vim.diagnostic                                    *diagnostic-api*
       • {severity}?   (`vim.diagnostic.SeverityFilter`) See
                       |diagnostic-severity|.
 
-*vim.diagnostic.GotoOpts*
+*vim.diagnostic.JumpOpts*
     Extends: |vim.diagnostic.GetOpts|
 
     Configuration table with the following keys:
 
     Fields: ~
-      • {cursor_position}?  (`{[1]:integer,[2]:integer}`, default: current cursor position)
-                            Cursor position as a `(row, col)` tuple. See
-                            |nvim_win_get_cursor()|.
+      • {diagnostic}?       (`vim.Diagnostic`) The diagnostic to jump to.
+                            Mutually exclusive with {count}, {namespace}, and
+                            {severity}. See |vim.Diagnostic|.
+      • {count}?            (`integer`) The number of diagnostics to move by,
+                            starting from {cursor_position}. A positive
+                            integer moves forward by {count} diagnostics,
+                            while a negative integer moves backward by {count}
+                            diagnostics. Mutually exclusive with {diagnostic}.
+      • {cursor_position}?  (`{[1]:integer,[2]:integer}`) Cursor position as a
```

Comment:
We are pretty consistent about using "pos" for "position". Is this a good time to rename this?

Reply:
40ba2f1c03a1e629383f0f78d91b59d10ddb4265